.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/python-3/.devcontainer/base.Dockerfile
+
+# [Choice] Python version: 3, 3.9, 3.8, 3.7, 3.6
+ARG VARIANT="3.9"
+FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
```

Comment:
Can we use a cuda-container here? Otherwise it can't be used for cuda-development and testing

Reply:
If you pass a gpu to the docker container on my windows machine it shows, that cuda is available. I'm not exactly sure, how this docker options works but can you verify, that it doesnt work on linux, if the host has cuda installed?

Reply:
There are two things here:

1.) You need a Nvidia-runtime for docker (that's out of the scope of this PR)
2.) There are images like https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch that have PyTorch with latest cuda and cudnn preinstalled. We could use one of these as base image

Reply:
This [image](https://catalog.ngc.nvidia.com/orgs/partners/teams/gridai/containers/pytorch-lightning/) also exists, but is not up to date. Would be even cooler, if we could use this. Can someone update it?

Reply:
just pushing a new v1.5.0 image...

Reply:
However, the current devcontainer images for vscode include some more default settings, which we would have to add. I'm currently trying to create a simple image based on the python image.

Reply:
@justusschock Do you think we can merge this without proper cuda support and deal with that in a follow up? Building a proper cuda devcontainer might take a little more time.

Reply:
I created https://github.com/microsoft/vscode-dev-containers/pull/1176 but merging might take a while.